README.md

```diff
@@ -25,7 +25,9 @@ var isChrome = require('browser-detection/is-chrome');
 var isIe9 = require('browser-detection/is-ie9');
 var isIe10 = require('browser-detection/is-ie10');
 var isIos = require('browser-detection/is-ios');
+var isIosUIWebview = require('browser-detection/is-ios-uiwebview');
```

Comment:
There's a section above this for how to use the methods on the main module. We should add these there as well.